### PR TITLE
pulp3-specific docs index for the docs builder

### PR DESCRIPTION
**Pull request by seandst**

### CODE REVIEW

**Comment by bmbouter** [comment]
@seandst Did you run the docs-builder script locally to test this or should I?

**Comment by bmbouter** [comment]
We could just merge and see I guess, it would probably be faster

**Comment by seandst** [comment]
Yeah, it totally failed. I forgot that I had this set up so I could actually test the docs build, so...I'ma do that and fix this up. :)

File: ci/docs/pulp_index.rst [hunk 1]
```diff
@@ -29,8 +29,6 @@ Content
    plugins/index
    dev-guide/index
    dev-guide/contributing/index
-   api/pulp/app/index
-   api/pulp/plugin
```
Comment (seandst) [inline ci/docs/pulp_index.rst]: These were introduced for pulp3, so they no longer need to be in this index.

**Comment by seandst** [comment]
Updated. I'm not sure how exactly `all_content_index` works, so let me know if what I did was silly and I'll fix it.

**Comment by bmbouter** [comment]
I think you need to git add `all_content_index_pulp3.rst`.

**Comment by seandst** [comment]
Missing file added

**Pull request merged**